profiles/seccomp/default_linux.go

```diff
@@ -552,6 +552,762 @@ func DefaultProfile() *Seccomp {
 				Caps: []string{"CAP_SYS_ADMIN"},
 			},
 		},
+		{
```

Comment:
This section allows `CLONE_NEWUSER` to be used if the Linux kernel is equal to or older than 4.18.

I did not think very hard about the choice of version. I only chose 4.18 because that is the version in which support for mounting FUSE filesystems in user namespaces was introduced.

Reply:
Should be 3.8, not 4.18
https://man7.org/linux/man-pages/man7/user_namespaces.7.html

Reply:
The intent was to only allow unprivileged user namespaces in more recent kernels, even though older kernels did support it, in the idea that it would be more secure. Maybe that was a bad idea.

Reply:
SGTM, but could you document that